portal/portal-sprint-103.md

```diff
@@ -0,0 +1,50 @@
+---
+slug: /release-notes/portal/sprint-101
+title: Sprint 101 (June 28, 2021)
+---
+
+# fiskaltrust.Portal - Sprint 103
+_June 28, 2021_
+
+## Features
+
+### Middleware Configuration
+
+- [Improved Queue SCU Page DE](#visual-improvements-to-middleware-component-dialogues)
+
+### Support
+
+- [Visualization of onboarded customer cases in portal](#visualization-of-onboarded-customer-cases-in-portal)
+
+## Middleware Configuration
+
+### Improved Queue SCU Page DE
+
+Some minor improvements of the QueueSCUDEConnectionList TableView (DE)  in Portal have been implemented. We invested some time in improving the way our tables are displayed. Visually nothing major has changed, aside from the styling of the Radio button, the addition of the title on the browser tab, and the addition of a Sorting option on the Outlet column. All previous functionalities are kept, but the consistency of all operations, as well as the performance of the page in different resolutions, has also been improved.
+
+## Support
+
+### Visualization of onboarded customer cases in portal
```

Comment:
I think we should add more detail when and how a case is added to this list.
A customer calls/sends an email request. 
We create a case internally, a case onboarding message is sent to the customer summarizing the case.
Email is sent out and case status is incremented, now a case is visible in the portal.

Reply:
The description I wrote was just for you Doxa to summarize what happens. Not to literally paste into the notes.
We should describe this process more from a customer's point of view.